src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -216,28 +216,28 @@ public static function get_theme_data( $deprecated = array() ) {
 	}
 
 	/**
-	 * Returns the custom post type that contains the user's origin config
+	 * Returns an array that contains global styles
 	 * for the current theme or a void array if none are found.
 	 *
-	 * This can also create and return a new draft custom post type.
+	 * This can also create and return a new draft post.
 	 *
 	 * @since 5.9.0
 	 *
 	 * @param WP_Theme $theme              The theme object. If empty, it
 	 *                                     defaults to the current theme.
-	 * @param bool     $should_create_cpt  Optional. Whether a new custom post
+	 * @param bool     $create_post        Optional. Whether a new custom post
 	 *                                     type should be created if none are
 	 *                                     found. Default false.
 	 * @param array    $post_status_filter Optional. Filter custom post type by
 	 *                                     post status. Default `array( 'publish' )`,
 	 *                                     so it only fetches published posts.
 	 * @return array Custom Post Type for the user's origin config.
 	 */
-	public static function get_user_data_from_custom_post_type( $theme, $should_create_cpt = false, $post_status_filter = array( 'publish' ) ) {
```

Comment:
Hey, the proposed naming `get_global_styles_from_post` doesn't reflect what this method does. If we want to remove the "cpt" reference I'd suggest using `get_user_data_from_{name of the cpt method}`.

Reply:
@oandregal What about renaming to `get_user_data_from_wp_global_styles`, i.e. where `wp_global_styles` is the custom type post?

Reply:
:+1: